upgrade-1.x-to-2.x.sh

```diff
@@ -563,6 +563,16 @@ umount /mnt/data
 umount /var/lib/docker
 umount /resin-data
 
+# Check whether /data has been correctly umounted, and stop if not, before anything's destroyed
```

Comment:
Should this be a function? Looks identical as the next check.

Reply:
You are right, definitely!